### PR TITLE
More explicit referencing of license of GeoStyler

### PR DESCRIPTION
**Pull request by marcjansen**
This links the `LICENSE` file from the `README` and adds a license header to all files ending in `{ts,tsx,less,js}`

### CODE REVIEW

**Comment by marcjansen** [comment]
BTW (and for future reference):

This was the helper script, which is not 100% failsafe (it removes `\\` in the files it finds). Maybe someone finds this useful nonetheless:

<details><summary>Hacky bash-code</summary>

```bash
#!/usr/bin/env bash

read -r -d '' HEADER << HEREDOC
/* Released under the BSD 2-Clause License
 *
 * Copyright (c) 2018-present, terrestris GmbH & Co. KG
 * All rights reserved.
 *
 * Redistribution and use in source and binary forms, with or without
 * modification, are permitted provided that the following conditions are met:
 *
 * * Redistributions of source code must retain the above copyright notice,
 *   this list of conditions and the following disclaimer.
 *
 * * Redistributions in binary form must reproduce the above copyright notice,
 *   this list of conditions and the following disclaimer in the documentation
 *   and/or other materials provided with the distribution.
 *
 * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
 * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
 * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
 * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
 * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
 * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
 * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 * POSSIBILITY OF SUCH DAMAGE.
 */

HEREDOC

cnt=0
changed=0
ignored=0

FILES=`find . -iname '*.ts' -or -iname '*.tsx' -or -iname '*.less' -or -iname '*.less'  -or -iname '*.js' | egrep -v "node_modules|dist|build"`
for FILE in $FILES
do
    cnt=$((cnt+1))
    line=`head -1 $FILE`
    firstHeader=`echo "${HEADER}" | head -1`
    if [ "$line" == "$firstHeader" ]; then
        ignored=$((ignored+1))
    else
        echo -e "$HEADER\n\n$(cat $FILE)" > $FILE
        changed=$((changed+1))
        #echo $FILE;
    fi
done;

echo "$cnt files visited, changed $changed, ignored $ignored."

```

</details>

**Comment by marcjansen** [comment]
Please carefully review, @jansule

File: README.md [hunk 1]
```diff
@@ -201,3 +201,7 @@ from within the GeoStyler Demo.
 ### <a name="developing-geostyler-data-parsers"></a>Developing GeoStyler Data Parsers
 
 Developing GeoStyler data parsers follows the same pattern as described in [Developing GeoStyler Style Parser](#Developing-GeoStyler-Style-Parsers), but keep in mind that **data parsers have to implement the [GeoStyler Data interface](https://github.com/geostyler/geostyler-data).**
+
+## <a name="license"></a>License
+
+GeoStyler is released under the BSD 2-Clause license. Please see see the file `LICENSE` in the root of this repository.
```
Comment (jodygarnett) [inline README.md:207]: You can use a markdown link here if you like:
```
[LICENSE.md](LICENSE.md)
```

Comment (jodygarnett) [reply README.md:207]: Oh I see the license is LICENSE, I just assumed LICENCE.md (from https://github.com/IQAndreas/markdown-licenses or similar).

File: src/Component/Symbolizer/PropTextEditor/PropTextEditor.tsx [hunk 2]
```diff
@@ -67,8 +95,8 @@ export class PropTextEditor extends React.Component<PropTextEditorProps> {
   }
 
   formatLabel = (label: string): string => {
-    const prefix = '\\{\\{';
-    const suffix = '\\}\\}';
+    const prefix = '\{\{';
+    const suffix = '\}\}';
```
Comment (jansule) [inline src/Component/Symbolizer/PropTextEditor/PropTextEditor.tsx:99]: Please restore the double `\` here, even though they probably can be removed entirely in a separate PR.
```suggestion
    const suffix = '\\}\\}';
```

File: README.md (hunk 1 above)
Comment (marcjansen) [reply README.md]: Done in 01cb65ccbb0965cf4e3ca5b943be5ba457bd03ce, thanks for the suggestion.

[the remark this answers is not in the document]
Comment (marcjansen) [reply README.md]: Has been fixed with 01cb65c, thanks for catching.

File: src/Component/Symbolizer/PropTextEditor/PropTextEditor.tsx (hunk 2 above)
Comment (marcjansen) [reply src/Component/Symbolizer/PropTextEditor/PropTextEditor.tsx]: Awesome catch! Done in 59b8865d6e5c768e8513bd8af213e77b9a977ae2

**Comment by marcjansen** [comment]
I have added the license header to md files in f092942aeb77da4af4ead14a658e57c34d7f197a.

@jansule it'd be nice if you could give this another round of review.

**Comment by marcjansen** [comment]
Thanks for the reviews and comments @jansule & @jodygarnett.

**Comment by jansule** [comment]
@samtux we added license headers to all project files. As you did substantial work on the spanish translations, feel free to add your name to the license header of https://github.com/geostyler/geostyler/blob/8816e684856970279c9558c8dd8b768c8ab03c9a/src/locale/es_ES.ts
